ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -20,12 +20,43 @@
 from abc import ABC, abstractmethod
 from typing import List, NamedTuple, Tuple, Optional, Union, Dict
 import numpy as np
-from enum import Enum
+from enum import Enum, IntEnum
 
 AgentId = int
 AgentGroup = str
 
 
+class EpisodeStatus(IntEnum):
+    """
+    The Episode is still ongoing
+    """
+
+    Default = (0,)
```

Comment:
Why are Default and Terminated tuples?

Reply:
Looks like my IDE did not want to comply on this. It is definitely a mistake.

Reply:
I don't think you need a comma between the enum values; it probably converted `1,` to that.
You might also be able to get rid of the `pylint` disables when that's fixed. If not, I want to remove pylint.